dora/core/client/fs/src/main/java/alluxio/client/file/DoraCacheFileSystem.java

```diff
@@ -382,11 +382,12 @@ public void rename(AlluxioURI src, AlluxioURI dst, RenamePOptions options)
     } catch (RuntimeException ex) {
       if (ex instanceof StatusRuntimeException) {
         Status.Code code = ((StatusRuntimeException) ex).getStatus().getCode();
-        if (code == Status.FAILED_PRECONDITION.getCode()) {
-          // throw new RuntimeException(ex.getMessage());
-        } else if (code == Status.NOT_FOUND.getCode()) {
+        if (Status.FAILED_PRECONDITION.getCode().equals(code)) {
+          LOG.error(String.format(
```

Comment:
What can be the precondition error here? Why don't we just throw an exception and exit?

Reply:
After reading the [grpc docs](https://grpc.github.io/grpc/core/status_8h.html#:~:text=(c)%20Use%20FAILED_PRECONDITION%20if%20the%20client%20should%20not%20retry%20until%20the%20system%20state%20has%20been%20explicitly%20fixed.) I think you are right. It has been changed.